app/locales/en/locationService.json

```diff
@@ -0,0 +1,13 @@
+{
+    "yes": "Yes",
+    "no": "No",
+    "location_enabled_title": "Private Kit Enabled",
+    "location_enabled_message": "Private Kit is securely storing your GPS coordinates once every five minutes on this device.",
```

Comment:
Please update these to `COVID Safe Paths` like in #331 , if you have any specific Q's about a specific translation, @penrods can help.